Chart.yaml

```diff
@@ -33,4 +33,9 @@ annotations:
       description: "Apply Security Context to all Pods"
       links:
         - name: "Related GitHub Issue"
```

Comment:
the artifacthub.io/changes: should only contain the latest change, in artifacthub it will create a history of each change.